pkg/util/flag/addr.go

```diff
@@ -0,0 +1,192 @@
+/*
```

Comment:
There is also an addr_test.go at https://github.com/projectatomic/atomic-enterprise/tree/master/pkg/cmd/flagtypes